Assets/HoloToolkit/Utilities/Scripts/Editor/EnforceEditorSettings.cs

```diff
@@ -11,23 +10,68 @@ namespace HoloToolKit.Unity
 
-            if (EditorSettings.serializationMode != SerializationMode.ForceText)
-            {
-                EditorSettings.serializationMode = SerializationMode.ForceText;
-                UnityEngine.Debug.Log("Setting Force Text Serialization");
-            }
+            if (TrueOnce()) {
+                if (EditorSettings.serializationMode != SerializationMode.ForceText)
+                {
+                    if (EditorUtility.DisplayDialog( 
+                            "Force Text Asset Serialization?", 
+                            "HoloToolkit is easier to maintain if the asset serialization mode for this project is set to \"Force Text\". Would you like to make this change?", 
+                            "Force Text Serialization", 
+                            "Later")) {
+                        EditorSettings.serializationMode = SerializationMode.ForceText;
+                        UnityEngine.Debug.Log("Setting Force Text Serialization");
+                    }
+                }
 
-            if (!EditorSettings.externalVersionControl.Equals("Visible Meta Files"))
-            {
-                EditorSettings.externalVersionControl = "Visible Meta Files";
-                UnityEngine.Debug.Log("Updated external version control mode: " + EditorSettings.externalVersionControl);
+                if (!EditorSettings.externalVersionControl.Equals("Visible Meta Files"))
+                {
+                    if (EditorUtility.DisplayDialog( 
+                        "Make Meta Files Visible?", 
+                        "HoloToolkit would like to make meta files visible so they can be more easily handled with common version control systems. Would you like to make this change?", 
+                        "Enable Visible Meta Files", 
+                        "Later")) {
+                        EditorSettings.externalVersionControl = "Visible Meta Files";
+                        UnityEngine.Debug.Log("Updated external version control mode: " + EditorSettings.externalVersionControl);
+                    }
+                }
             }
 
             #endregion
         }
+
+        /// <summary>
+        /// Returns true the first time it is called within this editor session, and
+        /// false for all subsequent calls.
+        /// </summary>
+        /// <remarks>
+        /// The Unity Editor does not provide a callback for when a project is opened.
+        /// InitializeOnLoad is triggered for all assembly reloads, including entering
+        /// and exiting PlayMode, and whenever a script is modified and recompiled.
+        ///
+        /// To ensure execution only when opening a project in a new instance of the editor,
+        /// store a timestamp in the editor key-value store whenever this function is called. 
+        /// The stored timestamp is then compared with the true start time of this editor
+        /// instance.
+        /// </remarks>
+        private static bool TrueOnce () 
```

Comment:
Can we rename this `IsNewEditorSession()` or `TrueOncePerEditorSession()` to make it more clear that its not `TrueOnceEver()`